p11/module.go

```diff
@@ -125,6 +125,26 @@ func (m Module) Slots() ([]Slot, error) {
 }
 
 // Destroy unloads the module/library.
-func (m Module) Destroy() {
+//
+// Once called, any code which uses this module might crash the application.
+func (m Module) Destroy() error {
+	modulesMu.Lock()
+	defer modulesMu.Unlock()
+
+	// Find initialized module based on ctx
+	var path string
+	for k, v := range modules {
+		if v.ctx == m.ctx {
+			path = k
+			break
+		}
+	}
+	if path == "" {
```

Comment:
is it really needed to return an error, or just make this a noop? With the later approach we don't need to change the method's signature

Reply:
I believe it works without it too. I've reverted the API change now :+1: